sigs.yaml

```diff
@@ -2363,21 +2363,23 @@ sigs:
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/OWNERS
 workinggroups:
-- dir: wg-apply
-  name: Apply
+- dir: wg-api-expression
+  name: API Expression
   mission_statement: >
-    Improve the state of declarative object management by fixing `kubectl apply`,
-    moving the logic from the client to the server. This move also enables new/fixed
-    features included in the scope of this working-group: diff, dry-run and prune.
-
-    Resources can be found in [this Google drive folder](https://drive.google.com/drive/folders/1wlpgkS2gFZXdp4x2WlRsfUBxkFlt2Gx0)
+    Improve API expressiveness and support API evolution.
```

Comment:
Mission Statement still needs work.

Reply:
Is it too much to copy everything in the proposal doc? :)

Reply:
If there's no other place to describe the purpose of the wg, clearly we should have it here. But I'm hoping somewhere we're supposed to describe what we're doing?

Reply:
OK let me think of a one-liner. Since we have a whole directory, maybe we could also commit a .md file there and link to it?

Reply:
How about: `Enable API authors to better serve API consumers, by improving, and documenting all aspects of Kubernetes API development. <link to expanded statement>`